MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentComponent/PaymentComponentBottomView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  PaymentComponentBottomView.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
```

Comment:
Could you please add one more line
//
//  PaymentComponentBottomView.swift
//  GiniMerchantSDK
//
//  Copyright © 2024 Gini GmbH. All rights reserved.
//